### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by wesk**
Create two issue templates to help ensure that contributors provide more useful information by default when opening a new issue.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (wistuba) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: pip freeze or syne tune version might be more useful

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```
Comment (wistuba) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]: lower case for consistency with other headings

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (wesk) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Good idea, added in next revision

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (wesk) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Updated in next revision

**Pull request merged**